### ISSUE TITLE
python3Packages.solo-python is broken

### ISSUE THREAD

**Issue by veprbl** [NixOS/nixpkgs#74835]
**Describe the bug**
See upstream issue solokeys/solo-python#49

**To Reproduce**
nix-build -A python3Packages.solo-python

**Expected behavior**
Package builds

**Metadata**
Please run `nix run nixpkgs.nix-info -c nix-info -m` and paste the result.

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
 - python3Packages.solo-python
# a list of nixos modules affected by the problem
module:
```

**Comment by veprbl** [comment]
cc @wucke13

**Comment by veprbl** [comment]
fixed in v0.0.21

**Comment by wucke13** [comment]
Should be fixed once #77611 gets merged
